src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -407,6 +407,12 @@
         - name: --message-of-the-day
           type: string
           short-summary: Path to a file containing the desired message of the day. Only valid for linux nodes. Will be written to /etc/motd.
+        - name: --enable-azure-keyvault-kms
+          type: bool
+          short-summary: (PREVIEW) Enable Azure KeyVault Key Management Service.
+        - name: --azure-keyvault-kms-key-id
```

Comment:
The linter reports:
```
-  FAIL - HIGH severity: option_length_too_long
    Parameter: aks create, `azure_keyvault_kms_key_id` - The lengths of all options ['--azure-keyvault-kms-key-id'] are longer than threshold 22. Argument azure_keyvault_kms_key_id must have a short abbreviation.
    Parameter: aks create, `enable_azure_keyvault_kms` - The lengths of all options ['--enable-azure-keyvault-kms'] are longer than threshold 22. Argument enable_azure_keyvault_kms must have a short abbreviation.
    Parameter: aks update, `azure_keyvault_kms_key_id` - The lengths of all options ['--azure-keyvault-kms-key-id'] are longer than threshold 22. Argument azure_keyvault_kms_key_id must have a short abbreviation.
    Parameter: aks update, `enable_azure_keyvault_kms` - The lengths of all options ['--enable-azure-keyvault-kms'] are longer than threshold 22. Argument enable_azure_keyvault_kms must have a short abbreviation.
```

Do we add a short abbreviation or use `--enable-keyvault-kms` to make the length less than 22?

Reply:
oh, you could suppress this warning in file `src/aks-preview/linter_exclusions.yml`.